WebApp/src/components/App.js

```diff
@@ -16,6 +17,7 @@ export default class App extends React.Component {
           <Route exact path="/" component={HomePage} />
           <Route path="/register" component={RegisterPage} />
           <Route path="/timetable" component={TimetablePage} />
+          <Route path="/studyGroup" component={StudyGroup} />
```

Comment:
we did not discussed this, but URLs should be written in kebab case `study-group`